### ISSUE TITLE
It doesn't work with this json, there should be a way to force formatting, even if it's not detected.

### ISSUE THREAD

**Issue by ofmarconi** [tulios/json-viewer#335]
https://localcine.com/wp-json/wp/v2/

**Comment by skamansam** [comment]
I agree - i would love the option to force formatting. For me, the only way to get this extension to work is if the file extension is '.json'. For django DRF projects, you can just specify a url like `mysite.com/path/to/json/?format=json` and it will give you json output, but `json-viewer` doesn't detect it at all.
